scanner/updater/export.go

```diff
@@ -56,42 +57,51 @@ func Export(ctx context.Context, outputDir string) error {
 		}
 	}()
 
+	var opts [][]updates.ManagerOption
+
+	// Manual CVEs.
 	manualSet, err := manual.UpdaterSet(ctx, nil)
 	if err != nil {
 		return err
 	}
-
 	outOfTree := append([]driver.Updater{}, manualSet.Updaters()...)
+	opts = append(opts, []updates.ManagerOption{updates.WithOutOfTree(outOfTree)})
+
+	// RHEL custom: Forked from ClairCore.
+	fac, err := rhel.NewFactory(ctx, rhel.DefaultManifest)
+	if err != nil {
+		return err
+	}
+	opts = append(opts, []updates.ManagerOption{
+		updates.WithFactories(map[string]driver.UpdaterSetFactory{"rhel-custom": fac})})
```

Comment:
I believe we'll need `updates.WithConfigs` as well so we can enable the unfixed vulns.

Reply:
[I thought it was ON by default](https://github.com/jvdm/quay-claircore-dev/blob/26b7d74e729c6e3850710b1f91934c4b883f92c9/rhel/rhel.go#L32).

Reply:
you're right i flipped the configuration around